ui/imports/utils/ProfileUtils.qml

```diff
@@ -45,17 +45,17 @@ QtObject {
         return "" // "custom" link type allows for user defined text
     }
 
-    function linkTypeColor(linkType) {
+    function linkTypeColor(linkType: int, palette: ThemePalette) : color {
```

Comment:
Maybe these colors should be also inside the palette?

Reply:
Yeah, would be great to invent better solution for that, especially that the palette is passed there only for handling default color. However I'm reluctant about making Theme aware of some external enums like that link type in this case.

Reply:
Could you create a task to follow-up this topic? 

I agree on your reasoning about `Theme. I thin the proposal should be just adding these external enums to the "static" `StatusColors` class.